SendValidationEmailActivity/handler.ts

```diff
@@ -0,0 +1,123 @@
+import * as t from "io-ts";
+
+import { isLeft } from "fp-ts/lib/Either";
+
+import * as HtmlToText from "html-to-text";
+import * as NodeMailer from "nodemailer";
+
+import { Context } from "@azure/functions";
+
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { EmailString } from "italia-ts-commons/lib/strings";
+
+import defaultHtmlTemplate from "io-functions-commons/dist/src/templates/html/default";
+import { sendMail } from "io-functions-commons/dist/src/utils/email";
+
+import { EmailDefaults } from "./";
+
+export function generateValidationEmailContentHtml(
+  emailValidationFunctionUrl: string,
+  token: string
+): string {
+  return `
+  <p>Click <a href="${emailValidationFunctionUrl}?token=${token}">here</a> to validate your email.</p>
```

Comment:
is this a stub ?

Reply:
Yes. I have added a comment and a story to define the real content.